plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -538,20 +538,32 @@ instance KnownBuiltinTypeIn DefaultUni term Integer => ReadKnownIn DefaultUni te
 
 instance UniOf term ~ DefaultUni => CaseBuiltin term DefaultUni where
     caseBuiltin (Some (ValueOf uni x)) branches = case uni of
+        -- TODO: It feels like we should support having only one branch to verify that a condition
```

Comment:
A gentle reminder that we've agreed that TODO/FIXME tags should be accompanied by a link to the corresponding github issue.